### PR TITLE
patron_types: display restrictions on detailed view

### PR DESCRIPTION
**Pull request by zannkukai**
Adapts the patron type resource detail view to include the display of
limits.

Authored-by: Renaud Michotte <renaud.michotte@gmail.com>

image

## Dependencies

https://github.com/rero/rero-ils/tree/US1643-borrow-limits

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

**Review by iGormilhit — CHANGES_REQUESTED** [review]
Commit proposition:
```
patron_types: display restrictions on detailed view 

Adapts the patron type detailed view to display the circulation
restrictions.
```

File: projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html [hunk 1]
```diff
@@ -37,5 +37,76 @@ <h1 class="mb-3">{{ record.metadata.name }}</h1>
...
+              <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>General limit</dt>
+              <dd class="col">{{ record.metadata.limits.checkout_limits.general_limit }}</dd>
+              <dd class="w-100 m-0"></dd>
+              <ng-container *ngIf="record.metadata.limits.checkout_limits.library_limit">
+                <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>Library limit</dt>
+                <dd class="col">{{ record.metadata.limits.checkout_limits.library_limit }}</dd>
+                <dd class="w-100 m-0"></dd>
+              </ng-container>
+              <ng-container *ngIf="record.metadata.limits.checkout_limits.library_exceptions">
+                <dt class="offset-1 col-11 label-title"translate>Exceptions</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html:65]: Missing space before `translate`

File: projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html [hunk 2]
```diff
@@ -37,5 +37,76 @@ <h1 class="mb-3">{{ record.metadata.name }}</h1>
...
+          <div class="card-header">
+            <i class="fa" [ngClass]="{
+              'fa-circle text-success': record.metadata.limits.checkout_limits,
+              'fa-circle-thin': !record.metadata.limits.checkout_limits
+            }"></i>
+            <span class="pl-3" translate>Limit by checkouts</span>
+          </div>
+          <div class="card-body" *ngIf="record.metadata.limits.checkout_limits">
+            <dl class="row m-0">
+              <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>General limit</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html:56]: Missing space before `translate`

File: projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html [hunk 3]
```diff
@@ -37,5 +37,76 @@ <h1 class="mb-3">{{ record.metadata.name }}</h1>
...
+              <dd class="col">{{ record.metadata.limits.checkout_limits.general_limit }}</dd>
+              <dd class="w-100 m-0"></dd>
+              <ng-container *ngIf="record.metadata.limits.checkout_limits.library_limit">
+                <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>Library limit</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html:60]: Missing space before `translate`

File: projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html [hunk 4]
```diff
@@ -37,5 +37,76 @@ <h1 class="mb-3">{{ record.metadata.name }}</h1>
...
+          <div class="card-header">
+            <i class="fa" [ngClass]="{
+              'fa-circle text-success': record.metadata.limits.fee_amount_limits,
+              'fa-circle-thin': !record.metadata.limits.fee_amount_limits
+            }"></i>
+            <span class="pl-3" translate>Limit by fee amount</span>
+          </div>
+          <div class="card-body" *ngIf="record.metadata.limits.fee_amount_limits">
+            <dl class="row m-0">
+              <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>Limit</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html:88]: Missing space before `translate`

File: projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html [hunk 5]
```diff
@@ -37,5 +37,76 @@ <h1 class="mb-3">{{ record.metadata.name }}</h1>
...
+          <div class="card-header">
+            <i class="fa" [ngClass]="{
+              'fa-circle text-success': record.metadata.limits.overdue_items_limits,
+              'fa-circle-thin': !record.metadata.limits.overdue_items_limits
+            }"></i>
+            <span class="pl-3" translate>Limit by overdue items</span>
+          </div>
+          <div class="card-body" *ngIf="record.metadata.limits.overdue_items_limits">
+            <dl class="row m-0">
+              <dt class="offset-1 col-lg-2 col-sm-4 label-title"translate>Limit</dt>
```
Comment (Garfield-fr) [inline projects/admin/src/app/record/detail-view/patron-types-detail-view/patron-types-detail-view.component.html:104]: Missing space before `translate`

**Review by Garfield-fr — APPROVED** [review]

**Review by iGormilhit — APPROVED** [review]
Commit message approved.

**Pull request merged**